discover.go

```diff
@@ -19,7 +19,11 @@ package openid
 // used as both the Claimed Identifier and the OP-Local Identifier
 // when an OP Identifier is entered.
 func Discover(id string) (opEndpoint, opLocalID, claimedID string, err error) {
-	return discover(id, urlGetter)
+	return discover(id, defaultInstance.urlGetter)
+}
+
+func (oid *OpenID) Discover(id string) (opEndpoint, opLocalID, claimedID string, err error) {
```

Comment:
I feel like at this point, we can get rid of the discover function below, and instead have Discover(id string) above just call defaultInstance.Discover(id). Same thing for the other similar functions RedirectURL and Verify.